### ISSUE TITLE
[UX] Add ability to only show aliased deployments

### ISSUE THREAD

**Issue by cryptiklemur** [FabienGreard/RequestWatcher#1]
Would be nice to only show aliased deployments, and if theres only the one deployment for that projectId, immediately move to the overview

**Comment by FabienGreard** [comment]
Yeah, there is without doubt some adjustment that could be done with the way we display deployments !
